### DISCUSSION TITLE
cannot rename existing organization to name of organization I deleted

### DISCUSSION THREAD

**Discussion by flyingmachine** [community/community#37593] [Repositories] [answered]
I have a github organization named `donut-power`. I wanted to rename it to `donut-party`. To do that I created a new organization, `donut-party`, planning to transfer all the repos over. However, I then saw that you could simply rename the organization. I deleted the `donut-party` organization and then tried to rename `donut-power` to `donut-party`. However, I'm now getting a message saying that the name `donut-party` is unavailable.

Is there any way to address this? Thanks!

**Comment by DSDmark** [comment, marked as answer]
Hey @flyingmachine,

Try to rename your organization after 90 days later because of **GitHub official** documentation it said:-

>  Note: After an account is deleted, the username at the time of deletion becomes unavailable for reuse for 90 days. To reuse an organization's username immediately, you must change the username before you delete the organization.

Check this out:-

[Deleting Organization](https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account)

**Comment by DSDmark** [comment]
And also I think you should read this which is written by Mr. @jdevstatic

> That's quite tricky, maybe try Support first, but there is no assurance
https://support.github.com/contact/account
maybe they will restore them for you, releasing usernames are tricky, we had several discussions of that in the past, also, the email will be a possible problem too, say, if you delete the Org and were not able to retrieve the email as being used by the Org, that will lose all the commits associated with it and the badges
but if you don't have much activity in that old account, you can simply keep your new account and never make the same mistake again, anyway, creating Orgs from your personal accounts are harmless,

https://github.com/community/community/discussions/36406#discussioncomment-3897838

**Reply by flyingmachine** [reply]
Wow this is very helpful, thank you!
